libs/juno-ui-components/src/docs/ColorPalette/TailwindThemeColors.js

```diff
@@ -0,0 +1,15 @@
+module.exports={
```

Comment:
Can you generate the different types (colors, theme colors, text colors) into the same file rather than having three different files? If not then please here also the comment at the start that explains that this is a generated file.